src/lbaf/Applications/LBAF_app.py

```diff
@@ -462,10 +462,6 @@ def run(self):
                     self.__logger, phase_id, reader=reader)
                 phase.populate_from_log(phase_id)
                 phases[phase_id] = phase
-                phase = Phase(
```

Comment:
That duplicate block is also deleted in PR #451 -- this may cause a conflict or not, we shall see